### PR TITLE
fix: store username in session storage upon login

### PR DESCRIPTION
**Pull request by isabellepakyishan**
Change requested by @Kuan-Zher - to return username to be stored in session storage

### CODE REVIEW

File: microservices/auth/auth.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ type Accounts struct {
 type Response struct {
 	ID       		int    	`json:"ID"`
 	Name     		string 	`json:"Name"`
+	Username      	string  `json:"Username"`
```
Comment (jiachenyee) [inline microservices/auth/auth.go]: this is slightly misaligned

**Review by jiachenyee — CHANGES_REQUESTED** [review]

File: microservices/auth/auth.go (hunk 1 above)
Comment (Kuan-Zher) [reply microservices/auth/auth.go]: agreed

**Review by Kuan-Zher — CHANGES_REQUESTED** [review]

File: microservices/auth/auth.go (hunk 1 above)
Comment (isabellepakyishan) [reply microservices/auth/auth.go]: made the change~

**Review by jiachenyee — APPROVED** [review]
perfect! lgtm!

**Review by Kuan-Zher — APPROVED** [review]
Looks good thanks for change

**Pull request merged**